Makefile

```diff
@@ -74,6 +74,9 @@ vendorcheck:
 .PHONY: install
 install: $(SOURCES)
 	go install -ldflags="$(LDFLAGS)" ./cmd/crc
+ifeq ($(GOOS),linux)
+	sudo setcap CAP_NET_BIND_SERVICE=+eip $(shell go env GOPATH)/bin/crc
```

Comment:
is this a necessity on ther users machine too to perform?

Reply:
Good question, from what I understand, the flag is preserved if we tar/untar the binary. It maybe deserves a preflight check?

Reply:
This is necessary to be able to bind ports < 1024

Reply:
Peopl;e do many different things to our tarball. We can not rely on preserving this I guess. A check might always be neater than a fail ;-)

Reply:
`make cross` and `make release` do not perform the `make install` target, so this will not be part of the release binary and tarball.